content/md/articles/cookbooks/date_and_time.md

```diff
@@ -1,13 +1,277 @@
-{:title "Date and Time (Help wanted)"
+{:title "Date and Time"
  :layout :page :page-index 4300}
 
-## Help wanted
+This cookbook covers working with Java's `java.time` package in two ways, the
+first with interop and the second uses a library that wraps `java.time` called
+`clojure.java-time`.
 
-Please follow the [instructions](https://github.com/clojure-doc/clojure-doc.github.io/tree/source#how-to-contribute) on how to contribute and start writing over [here](https://github.com/clojure-doc/clojure-doc.github.io/blob/source/content/md/articles/cookbooks/date_and_time.md)
+This guide covers Clojure 1.11.1 and Java 8.
+
+# Introduction
+
+This cookbook does not assume a background in Java, if you have such a
+background the interop section might be of interest.
+
+This cookbook will be useful if you got your Clojure environment setup, know
+some of the basics and are looking to work with time and dates in Clojure.
+
+This guide aims to cover date and time in the JVM and JS domains.
+
+What's the scope of this cookbook?
+What will I cover and what will I not cover?
+
+This cookbook will cover working with `java.time` as `java.util.Date` is legacy[^1].
+
+## Overview
+
+The representation of date and time is dependent upon the host, for example in the JVM
+
+- Dates are a `java.util.LocalDate` object.
+- Time is a a `java.util.LocalTime` object.
+- Clojure can represent these objects as a `java.util.LocalDate`, `java.util.LocalTime`
+  and `java.util.LocalDateTime` respectively.
+- The `#inst` is a [tagged
+  literal](https://clojure.org/guides/weird_characters#tagged_literals) is used
+  to represent a `java.util.Date` object.
+
+While in JavaScript:
+TBD
+
+
+## Libraries
+
+* [clojure.java-time](https://github.com/dm3/clojure.java-time)
+  > A Clojure wrapper for Java 8 Date-Time API.
+* [cljc.java-time](https://github.com/henryw374/cljc.java-time)
+  > A Clojure(Script) library which mirrors the java.time api through kebab-case-named vars.
+* [tick](https://github.com/juxt/tick)
+  > A Clojure(Script) & babashka library for dealing with time. Intended as a replacement for clj-time.
+
+> **TODO** Should we recommend a lib?  *N* is the **recommended** library to use
```

Comment:
I don't think the cookbook should recommend a specific library here since the choice is very subjective. Probably worth adding a note to the description of `clojure.java-time` that it is what is used in this guide (although, if you are going to add `cljc.java-time` examples as well, that's probably not needed).

Reply:
I agree that I don't think the cookbook should recommend a specific library. My next step is adding `cljc.java-time` examples so I think it'll be clear that the cookbook uses two libraries.